### PR TITLE
Initial development

### PR DESCRIPTION
**Pull request by jbachhardie**
- Program to query the consul API and get back container names registered to the serviceID by registrator
- https://trello.com/c/jI3lE7Tx/339-ci-develop-a-tool-that-gets-all-containers-for-a-given-service-tag

### CODE REVIEW

File: src/cli.js [hunk 1]
```diff
@@ -8,10 +8,12 @@ program
   .version(version);
 
 program
-  .option('-s, --service <name>', 'The name of the service to search for.')
-  .option('-t, --tag <tag>', 'The tag of the service to search for.')
-  .option('-h, --host <hostname>', 'The hostname of the docker host which hosts the service.')
-  .option('-c, --consul <hostname:port>', 'The hostname and port of the consul server to search.')
-  .parse(process.argv);
+  .arguments('<consul-host> <service-host> <service-name> <service-tag>')
+  .option('-p, --port <port>', 'Port to query the consul host on', parseInt)
```
Comment (sashman) [inline src/cli.js]: What happens if the port isn't given? Could we default to 8500?

Comment (jbachhardie) [inline src/cli.js]: We do (or rather, npm-consul does)

Comment (sashman) [inline src/cli.js]: Could we update the description to make that clear please?

**Comment by sashman** [comment]
:+1:

**Pull request merged**